### ISSUE TITLE
Auto-update of AdGuard Home

### ISSUE THREAD

**Issue by talonric332** [AdguardTeam/AdGuardHome#5728]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Description

(feature request) Add a feature in the settings part of the dashboard that users can enable that will auto update AdGuard home as soon as a new update is available, so you wouldn’t have to go into the dashboard. Can you guys also add a “total domains/sites blocked with filters” box to the dashboard, similar to what pi-hole has?

**Comment by ainar-g** [comment]
Please only file a single feature request per issue.

 >  Can you guys also add a “total domains/sites blocked with filters” box to the dashboard, similar to what pi-hole has?

There is already a “Blocked by filters” graph on the dashboard.  Or is that not what you're looking for?

**Comment by talonric332** [comment]
I’m looking for something that tells you the total amount of filtering rules combined from all of your filters and any other custom filtering rules you have

**Comment by ainar-g** [comment]
A, I see.  That sounds like #5316.

**Comment by KaKi87** [comment]
Hello,
Any news on this ?
Thanks

**Comment by ainar-g** [comment]
Merging into the aforementioned issue.

**Comment by KaKi87** [comment]
No, I'm talking about the first feature request, the one this issue is entitled with, and its description starts with. Thanks

**Comment by ainar-g** [comment]
@KaKi87, the title sounds like “auto-update of AdGuard Home, the program”, and that had already been implemented long ago.  The feature request about the dashboard auto-update is #1739.

**Comment by KaKi87** [comment]
Please take the time to read the description that OP took the time to write :

> Add a feature in the settings part of the dashboard that users can enable that will auto update AdGuard home as soon as a new update is available, so you wouldn’t have to go into the dashboard.

Thanks

**Comment by ainar-g** [comment]
I got confused by multiple feature requests in the OP, sorry.  Reopening and retitling.

**Comment by VsevolodNaumov** [comment]
Thanks for taking the time to report this.
I believe this is already being tracked in #4448, so I'm going to close this issue as a duplicate to keep all context in one thread. If you notice anything that is not covered there, please leave a comment in #4448 and we'll take it into account.

**Comment by KaKi87** [comment]
"on (re)starts" isn't the same as "immediately".
